Example/GiniVision/Info.plist

```diff
@@ -49,7 +49,7 @@
 	<key>CFBundlePackageType</key>
 	<string>APPL</string>
 	<key>CFBundleShortVersionString</key>
-	<string>0.8.2</string>
+	<string>0.8.3</string>
```

Comment:
You have to remember to change the version in the multipage branch then. It's also 0.8.3